architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -175,17 +175,58 @@ endif::[]
+the image stream will still point to a known-good version of the image, ensuring that your application will not break unexpectedly.
+
+Note that an image stream can be configured to periodically chack for updates (using the `--scheduled` flag on a tag) 
+or you can re-importing an image (using the `oc import-image` command). 
+
+An image stream does not contain actual image data, it is simply a mapping between tags and actual images. 
+It can contain additional metadata that describes the tags provided by the image stream.
+
+The images can be stored in any of the following:
+
+. {product-title}'s xref:../../architecture/infrastructure_components/image_registry.adoc#integrated-openshift-registry[integrated
+registry]
+. An external registry, such as `registry.access.redhat.com`, `hub.docker.com`, and so forth.
+. Other image streams
+
 An _image stream_ comprises any number of
 xref:../../architecture/core_concepts/containers_and_images.adoc#docker-images[Docker-formatted
 container images] identified by tags. It presents a single virtual view of
-related images, similar to an image repository, and may contain images from any
-of the following:
+related images, similar to an image repository. 
 
-. Its own image repository in {product-title}'s
-xref:../../architecture/infrastructure_components/image_registry.adoc#integrated-openshift-registry[integrated
-registry]
-. Other image streams
-. Image repositories from external registries
+==== Important terms 
+
+Docker repository::
+A central location for uploading and downloading images.
+For example, the OpenShift Jenkins images are in a Docker registry: *docker.io/openshift/jenkins-2-centos7:latest*.
+
+Docker image tag::
+A  label applied to a Docker image in a repository that distinguish specific images. For example, here *latest* is a tag:
+*docker.io/openshift/jenkins-2-centos7:latest*
+
+Docker image sha::
+A SHA code that can be used to pull an image. For example:
+*docker.io/openshift/jenkins-2-centos7@sha256:ab312bda324*.
+
+Image stream::
+An {product-title} object that contains pointers to any number of Docker-formatted container images identified by tags. 
+The pointers are called _image stream tags_. See xref:image-streams[Image Streams] below.
+
+Image stream tag::
+A named pointer to an image in an image stream. See xref:image-stream-tag[Image Stream Tag] below.
+
+Image stream image::
+An image that allows you to retrieve a specific Docker image from a particular image stream where it is tagged. 
+See xref:image-stream-image[Image Stream Images] below.
+
+Image stream trigger::
+A trigger causes a specific action when an image stream tag changes. Importing an image can be a trigger, along with other things. An import only causes a trigger if an image stream is following the associated image stream tag. See xref:image-stream-triggers[Image Stream Triggers] below.
```

Comment:
Import causes a trigger to fire *only* when there are deployments, builds or other resources listening for those.